subsys/net/lib/mqtt_helper/Kconfig

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MQTT_HELPER
+	bool "MQTT helper library"
+	select NET_SOCKETS_POSIX_NAMES if !POSIX_API
+	select MQTT_LIB
+	help
+	  Convenience library that simplifies Zephyr MQTT API handling and socket polling.
+
+if MQTT_HELPER
+
+config MQTT_HELPER_NATIVE_TLS
+	bool "Native TLS socket"
+	help
+	  Enabling this option will configure the socket to be native for TLS
+	  instead of offloading TLS operations to the modem.
```

Comment:
It's not necessarily to the modem, it can also be some other device's own networking stack

Reply:
Something like this?

```
	  Enabling this option will configure the socket to be native for TLS
	  instead of offloading TLS operations to the device's networking stack.
```

Reply:
Yeah, maybe that's better. Probably just the word `native` that always confuses me, it's not exactly self-describing what it means.

Reply:
Agree, its definitely confusing